subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolveConfigurationDependenciesBuildOperationIntegrationTest.groovy

```diff
@@ -258,7 +263,79 @@ class ResolveConfigurationDependenciesBuildOperationIntegrationTest extends Abst
         "script plugin" | 'buildscript' | "scriptPlugin.gradle"
         "settings"      | 'buildscript' | 'settings.gradle'
         "init"          | 'initscript'  | 'init.gradle'
+    }
+
+    def "included build classpath configuration resolution result is exposed"() {
```

Comment:
Why aren't we using the standard build operation observation fixtures?

Reply:
We don't use the standard operation fixture here because when using the standard build operation fixture, operation events are directly serialized to disk and we don't mimic the delay between having the build operations fired (in an included build) and reading the build operation from a details/result from the root build when the resolution store of the included build was already cleaned up.

Reply:
Going to open a separate issue to get that changed.